scripts/preload_models.py

```diff
@@ -36,6 +40,19 @@
 transformer = CLIPTextModel.from_pretrained(version)
 print('...success')
 
+
+print('preloading Stable Diffusion model file...', end='')
+sd1_4 = Path(hf_hub_download("CompVis/stable-diffusion-v-1-4-original", "sd-v1-4.ckpt"))
+# inpainting = Path(hf_hub_download("runwayml/stable-diffusion-inpainting", "sd-v1-5-inpainting.ckpt"))
+# sd1_5 = Path(hf_hub_download("runwayml/stable-diffusion-v1-5", "v1-5-pruned.ckpt"))
+
+model_dest = Path('models/ldm/stable-diffusion-v1/model.ckpt')
+model_dest.parent.mkdir(parents=True, exist_ok=True)
+sd1_4.resolve(strict=True).link_to(model_dest)
```

Comment:
It would be nice to uncomment the 1.5 download line so that the user gets both 1.4 and 1.5 installed, and provide a command-line switch that enables the user to select one or both.